Sources/ReplyService/Activity+ReplyRequest.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Activity+ReplyRequest.swift
+//  SwiftBot
+//
+//  Created by Andrew Tokarev on 27/06/2017.
+//
+//
+
+import Foundation
+import BotsKit
+
+extension Activity : ReplyRequest {
```

Comment:
I mean that we can have different providers to send messages, and Activity is already generic type. I think better to transform Activity in some already provider specified type